controller/test-files/deployments_osio/namespace-by-type.json

```diff
@@ -0,0 +1,44 @@
+{
+  "data": {
+    "attributes": {
+      "bio": "",
+      "cluster": "https://some-random-cluster-dot.com",
+      "company": "",
+      "contextInformation": {
+        "experimentalFeatures": {
+          "enabled": true
+        },
+        "recentContexts": [
+          {
+            "space": "00000000-0000-0000-0000-000000000000",
+            "user": "11111111-1111-1111-1111-111111111111"
```

Comment:
By looking at the UUIDs I would say that you have handcrafted this file. If you run a test individually and pass an `-update` switch (e.g. `go test ./controller -run MyTestSuite -testify.m MyTest -update`) every `compareWithGoldenUUIDAgnostic()` call will first write out the file before comparing it. This eliminates the need for handcrafted golden files.